packages/webapp/src/containers/Finances/RevenueDetail/index.jsx

```diff
@@ -109,18 +68,16 @@ function SaleDetail({ history, match }) {
     history.back();
   };
 
-  const onTypeChange = (typeId, setValue, REVENUE_TYPE_ID) => {
+  const onTypeChange = (typeId, setValue, REVENUE_TYPE_OPTION) => {
```

Comment:
You don't need to update it, but `REVENUE_TYPE_OPTION` is just a parameter and not a constant, so I think it is usually camelCase.

Reply:
removed it and imported the constant.